### DISCUSSION TITLE
support to generat test data

### DISCUSSION THREAD

**Discussion by EurekaJiang** [dbeaver/dbeaver#18260] [Ideas]
I used to be a java developer with bank enviroment. And it happens often that developers  generate the test data  in the software testing life cycle , with several tables and  some columns with specific value. Sometimes  I use the program to generate the sql script , but it also needs me to spell the columns.   I wonder if there can be a new feature in DBeaver that users can choose the tables and  columns   to generat testdata。Most developer use the DBeaver in working , I think both of the developer and the  testing engineer will benefit  a lot  from it .

**Comment by E1izabeth** [comment]
DBeaver has this feature in PRO version. See https://dbeaver.com/docs/wiki/Mock-Data-Generation-in-DBeaver/

**Reply by EurekaJiang** [reply]
yeah,  but I mean it may support serveral tables related at the same time.

**Reply by E1izabeth** [reply]
There is no such option yet. You could create a feature request issue if you want.
